tools/clang/unittests/HLSL/DxilContainerTest.cpp

```diff
@@ -1762,6 +1763,274 @@ TEST_F(DxilContainerTest, DxcUtils_CreateReflection) {
   }
 }
 
+TEST_F(DxilContainerTest, CheckReflectionQueryInterface) {
+  if (m_ver.SkipDxilVersion(1, 5))
```

Comment:
Could we get a comment explaining why this version cutoff is needed?

Reply:
This was copied from another reflection test, but this one can probably get away with a min version of 1.3 (for library support).  I'm updating it, along with a comment.